src/components-examples/material/tabs/tab-group-align/tab-group-align-example.html

```diff
@@ -1,18 +1,18 @@
 <!-- #docregion align-start -->
-<mat-tab-group mat-align-tabs="start">
+<mat-tab-group mat-stretch-tabs="false" mat-align-tabs="start">
```

Comment:
Is `mat-stretch-tabs` documented?

Reply:
Yes, `mat-stretch-tabs` has a javadoc style documentation that surfaces on https://material.angular.io/components/tabs/api.

The behavior of mat-stretch-tabs changes a little bit for the MDC-based tabs. We can add that when we get to documenting migrations

Tab header labels default to stretching the width of the container. This can be turned off by providing mat-stretch-tabs="false".